x-pack/plugins/ingest_manager/server/services/epm/registry/requests.ts

```diff
@@ -4,20 +4,26 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import Boom from 'boom';
 import fetch, { Response } from 'node-fetch';
 import { streamToString } from './streams';
+import { IngestManagerError, IngestManagerErrorType } from '../../../errors';
 
 export async function getResponse(url: string): Promise<Response> {
   try {
     const response = await fetch(url);
     if (response.ok) {
       return response;
     } else {
-      throw new Boom(response.statusText, { statusCode: response.status });
+      throw new IngestManagerError(
+        IngestManagerErrorType.RegistryError,
+        `Error connecting to package registry: ${response.statusText}`
```

Comment:
Can we have the url in the message too? I had some issue in jenkins where it's not clear what happens

Reply:
Sure.

Reply:
In certain error cases we'll have the url twice in the error message now, because when `fetch` throws an error that contains the url as well.

It's better than not having it at all I guess.